views/404-page.pug

```diff
@@ -0,0 +1,21 @@
+doctype html
+html
+  head
+      block head
+        title="Never Forget"
+        link(rel='stylesheet' href='/stylesheets/reset.css')
+        link(rel='stylesheet' href='/stylesheets/navbar.css')
+        link(rel="stylesheet", href='/stylesheets/404.css')
+        script(src="https://kit.fontawesome.com/e145510635.js" crossorigin="anonymous")
+
+
+  body
+    block content
+      nav.navbar
```

Comment:
@dezadkins I think the a needs another space in front of it and then the img needs to be indented under the a